### DISCUSSION TITLE
OpenStreetMap Search field

### DISCUSSION THREAD

**Discussion by CarstenOppermann** [directus/directus#18031] [Archive]
### Summary

Would it be possible to integrate an Address Search field  in the Map-Interface of  a geopational data field?

### Motivation

needed for improvement of geocoding an event address for users.

### Detailed Design

as it is in OpenStreetMap

### Requirements List

B

### Drawbacks

implementation cost, both in term of code size and complexity

### Alternatives

manually search the point in the map

### Adoption Strategy

-

**Comment by rijkvanzanten** [comment]
Archiving this, as the template hasn't been filled out properly
